iamchrissmith/uncorked
# Twilio setup [WIP]

#### What does this PR do?

This implements Twilio two-factor authorization via SMS upon creation of a new user account

#### Where should the reviewer start?

There are 13 changed files -- it's probably best to look over all of them then follow the instructions below

#### How should this be manually tested?

* Spin up the server
* Visit localhost:3000/users/new in your browser
* Create a new user using your real phone number
* Follow the instructions on the screen to verify and complete the new user creation

#### Any background context you want to provide?

I haven't figured out how to mock the SMS yet so this is untested. This also breaks our guest registration feature test, thus it shouldn't be merged into master yet.

#### What are the relevant tickets?

https://www.pivotaltracker.com/story/show/148677283

#### Questions:

  - Do Migrations Need to be ran?

Yes

  - Do Environment Variables need to be set?

 Yes -- check Slack

  - Any other deploy steps?

 No
